### PR TITLE
adding code

### PR DESCRIPTION
**Pull request by reddydevops2000**
Hii Vijay,

i have added my code please review the code.

regards,
Reddy

### CODE REVIEW

File: ansible.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ ansible playbook for java package
 -host:localhost
  tasks:
 -name:install java
-      - java
+      - maven
```
Comment (vijay990855) [inline ansible.txt:7]: add maven version

File: terraform.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 aws resource "aws_instance= "demo-server"{
 key_pair= "centos"
 ami="ami-8k60ke53jde8tbn"
-instace_type="t2-nano"
+instace_type="t2-micro"
```
Comment (vijay990855) [inline terraform.txt:4]: change instance type to t2-large

**Review by vijay990855 — COMMENTED** [review]
added review comments please work

File: terraform.txt (hunk 2 above)
Comment (reddydevops2000) [reply terraform.txt:4]: changed t2.large

File: ansible.txt (hunk 1 above)
Comment (reddydevops2000) [reply ansible.txt:7]: added  maven version

**Review by vijay990855 — APPROVED** [review]
looks good

**Pull request merged**